projects/1.18/assets/extra-utilities-reborn/extrautilitiesrebirth/lang/zh_cn.json

```
@@ -0,0 +1,173 @@
+  "block.extrautilitiesrebirth.compressed_gravel": "压缩沙砾",
+  "block.extrautilitiesrebirth.compressed_sand": "压缩沙子",
+  "block.extrautilitiesrebirth.creative_chest": "创造模式箱子",
+  "block.extrautilitiesrebirth.creative_drum": "创造桶",
+  "block.extrautilitiesrebirth.creative_energy_storage": "创造模式能量源",
+  "block.extrautilitiesrebirth.creative_harvest": "创造模式收割者",
+  "block.extrautilitiesrebirth.creative_inv_sample": "创造库存样本",
+  "block.extrautilitiesrebirth.creative_spikes": "创造锥刺",
+  "block.extrautilitiesrebirth.creeper_glass": "苦力怕玻璃",
+  "block.extrautilitiesrebirth.culinary_generator": "厨用发电机",
+  "block.extrautilitiesrebirth.cursed_earth": "诅咒之土",
+  "block.extrautilitiesrebirth.dark_ethereal_glass": "暗色天域玻璃",
+  "block.extrautilitiesrebirth.dark_glass": "黑暗玻璃",
+  "block.extrautilitiesrebirth.death_generator": "死亡发电机",
+  "block.extrautilitiesrebirth.diamond_edged_computational_matrix": "钻石蚀刻计算矩阵",
+  "block.extrautilitiesrebirth.diamond_spike": "钻石锥刺",
+  "block.extrautilitiesrebirth.disenchantment_generator": "祛魔发电机",
+  "block.extrautilitiesrebirth.double_compressed_cobblestone": "双重压缩圆石",
+  "block.extrautilitiesrebirth.double_compressed_dirt": "双重压缩泥土",
+  "block.extrautilitiesrebirth.double_compressed_gravel": "双重压缩沙砾",
+  "block.extrautilitiesrebirth.double_compressed_sand": "双重压缩沙子",
+  "block.extrautilitiesrebirth.edged_glass": "镶边玻璃",
+  "block.extrautilitiesrebirth.edged_stone_bricks": "镶边石砖",
+  "block.extrautilitiesrebirth.enchanter": "附魔器",
+  "block.extrautilitiesrebirth.ender_core": "末影之核",
+  "block.extrautilitiesrebirth.ender_generator": "末影发电机",
+  "block.extrautilitiesrebirth.ender_infused_obsidian": "末影黑曜石",
+  "block.extrautilitiesrebirth.ender_lilly_endstone": "末影百合末地石",
+  "block.extrautilitiesrebirth.ender_lilly_stage_1": "末影百合种子",
+  "block.extrautilitiesrebirth.ender_lilly_stage_2": "末影百合台阶",
+  "block.extrautilitiesrebirth.ender_lilly_stage_3": "末影百合",
+  "block.extrautilitiesrebirth.ender_lilly_stage_4": "末影百合",
+  "block.extrautilitiesrebirth.ender_lilly_stage_5": "末影百合",
+  "block.extrautilitiesrebirth.ender_lilly_stage_6": "末影百合",
+  "block.extrautilitiesrebirth.ender_lilly_stage_7": "末影百合",
+  "block.extrautilitiesrebirth.ender_lilly_stage_8": "末影百合",
+  "block.extrautilitiesrebirth.ender_marker": "末影路标",
+  "block.extrautilitiesrebirth.ender_quarry": "末影采石场",
+  "block.extrautilitiesrebirth.ender_quarry_silk_touch_upgrade": "末影采石场精准升级",
+  "block.extrautilitiesrebirth.ender_quarry_silk_touch_upgrade_2": "末影采石场精准II升级",
+  "block.extrautilitiesrebirth.ender_quarry_silk_touch_upgrade_3": "末影采石场精准III升级",
+  "block.extrautilitiesrebirth.ender_quarry_speed_upgrade": "末影采石场效率I升级",
+  "block.extrautilitiesrebirth.ender_quarry_upgrade_base": "末影采石场升级基座",
+  "block.extrautilitiesrebirth.ender_quarry_world_hole_upgrade": "末影采石场世界坑升级",
+  "block.extrautilitiesrebirth.ender_thermic_pump": "末影熔岩泵",
+  "block.extrautilitiesrebirth.ethereal_glass": "天域玻璃",
+  "block.extrautilitiesrebirth.fe_transmitter": "Fe传输器",
```

```suggestion
  "block.extrautilitiesrebirth.fe_transmitter": "FE传输器",
```
这里是Forge Energy的缩写，均需大写